guides/source/active_storage_overview.md

```diff
@@ -983,6 +1059,19 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 end
 ```
 
+If you're using [parallel tests][], you should also configure each process to use its own
+folder for Active Storage:
+
+```ruby
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  # ...
+  parallelize_setup do |i|
+    ActiveStorage::Blob.service.instance_variable_set(:@root, "#{ActiveStorage::Blob.service.root}-#{i}")
```

Comment:
`root` is an `attr_reader`: https://github.com/rails/rails/blob/main/activestorage/lib/active_storage/service/disk_service.rb#L12

Making it an `attr_accessor` would be helpful for this, but it might not be a smart thing to do in general.